tcms_tenants/context_processors.py

```diff
@@ -6,6 +6,8 @@ def tenant_navbar_processor(request):
     Provide tenant name for display in navbar
     """
     tenant_name = request.tenant.schema_name
+    if request.tenant.name:
+        tenant_name = request.tenant.name
```

Comment:
Should be after `tenant.organization`, see https://github.com/kiwitcms/tenants/pull/170/files/9aee1faaee36d73825fbf43da22b4775c3923a10..1796df1707c0fe99b6a4b0a49625aa8372bf1873#r917685230